automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/NotebookRKernelSpec.scala

```diff
@@ -101,6 +101,14 @@ class NotebookRKernelSpec extends ClusterFixtureSpec {
       }
     }
 
+    "should have tidyverse automatically installed" in { clusterFixture =>
+      withWebDriver { implicit driver =>
+        withNewNotebook(clusterFixture.cluster, RKernel) { notebookPage =>
+          notebookPage.executeCell("library(tidyverse)").get should not include("there is no package")
```

Comment:
Could we add a positive verification here? E.g. `should include X`.

The reason is that this could still throw an error meaning it's not working correctly, but still pass the test because it doesn't contain the string "there is no package".